src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int countBalls = 3;
+
+        for (int i = 0; i < countBalls; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
```

Comment:
[Don't create temporary variables if you can directly return value from the method](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)
```suggestion
            System.out.println(lottery.getRandomBall());
```